language/ir-testsuite/tests/transaction_fee_distribution/test_txn_fee_collection.mvir

```diff
@@ -5,13 +5,17 @@
 
 import 0x0.LibraAccount;
```

Comment:
Do we want to kill these old IR tests now that there are Move versions?

Reply:
I'm not sure if we're killing the IR tests after translating them? I'll take a look and do whatever the convention has been so far if that sounds good.

Reply:
Makes sense! Let's discuss in the Monday team meeting.